python/servo/post_build_commands.py

```diff
@@ -263,11 +263,16 @@ def package(self, release=False, dev=False, debug=False, debugger=None):
     @CommandArgument('--dev', '-d', action='store_true',
                      help='Package the dev build')
     def install(self, release=False, dev=False):
-        binary_path = self.get_binary_path(release, dev, android=True)
-        if not path.exists(binary_path):
-            # TODO: Run the build command automatically?
-            print("Servo build not found. Please run `./mach build` to compile Servo.")
-            return 1
+        try:
+            binary_path = self.get_binary_path(release, dev, android=True)
+        except BuildNotFound:
+            print("Servo build not found. Building servo...")
+            result = Registrar.dispatch(
+                "build", context=self.context, release=release, dev=dev
+            )
+            if result:
+                return result
+            self.install(release=release, dev=dev)
```

Comment:
Another option would be calling `binary_path = self.get_binary_path(release, dev, android=True)` again. I chose calling install again because maybe later on more checks will be added to this function, and we may want to run those after Servo has been built. If I go this route I'm going to need `return self.install(release=release, dev=dev)`.